buildsystem/spack/crusher/modules/exago.sh

```diff
@@ -1,3 +1,3 @@
-module use -a /lustre/orion/csc359/proj-shared/nkouk/spack-install/modules/linux-sles15-zen3
-# exago@=develop%clang@=14.0.0-rocm5.2.0-mixed~cuda+hiop~ipo+ipopt+logging+mpi~python+raja+rocm amdgpu_target=gfx90a build_system=cmake build_type=MinSizeRel dev_path=/lustre/orion/scratch/nkouk/csc359/ExaGO generator=make arch=linux-sles15-zen3
-module load exago/develop-clang-14.0.0-rocm5.2.0-mixed-h3wqolf
+module use -a /lustre/orion/eng145/proj-shared/nkouk/spack-install/modules/linux-sles15-zen3
+# exago@=develop%clang@=14.0.0-rocm5.2.0-mixed~cuda+hiop~ipo+ipopt+logging+mpi~python+raja+rocm amdgpu_target=gfx90a build_system=cmake build_type=Release dev_path=/lustre/orion/scratch/nkouk/eng145/ExaGO generator=make arch=linux-sles15-zen3
```

Comment:
Could we install these modules in "world shared" folder?

Reply:
Sure!

Reply:
I'll make a new pull request.